### PR TITLE
Add helper to convert 3 consecutive spaces to line-breaks

### PR DESCRIPTION
**Pull request by mmmmillar**
### WHAT:

Add a helper to convert groups of 3 spaces to line-breaks.

### WHY:

I want to be able to add line breaks in html headers in Contentful.

### EXAMPLE:

I want the title "Legacy Transformation Services" to have line-breaks so that the header html looks like `<h1>Legacy<br/>Transformation<br/>Services</h1>`
To add a header with line-breaks, I need to add groups of 3 spaces to the title. So the title I would add to Contentful would be "Legacy   Transformation   Services".

### CODE REVIEW

File: src/components/Contentful/Hero.js [hunk 1]
```diff
@@ -47,7 +48,7 @@ export default function ContentfulHero({
         <div className="row">
           <div className="col-7">
             {sectionNameComponent}
-            <h1 className="contentful-hero__page-title">{pageTitle}</h1>
+            <h1 className="contentful-hero__page-title" dangerouslySetInnerHTML={{__html: threeSpaceToLineBreak(pageTitle)}} />
```
Comment (lukemorton) [inline src/components/Contentful/Hero.js]: You should be able to define this function in the Hero.js file:

```js
function threeSpaceToLineBreak(pageTitle) {
  return pageTitle.split('   ').splice(1, 0, <br />)
}
```

Then use it like so:

```jsx
<h1 className="contentful-hero__page-title">{threeSpaceToLineBreak(pageTitle)}</h1>
```

Comment (lukemorton) [inline src/components/Contentful/Hero.js]: In order to not lose the safety of React, you could use something like this instead: https://github.com/iansinnott/react-string-replace

**Comment by mmmmillar** [comment]
> ### WHAT:
> Add a helper to convert groups of 3 spaces to line-breaks.
> 
> ### WHY:
> I want to be able to add line breaks in html headers in Contentful.
> 
> ### EXAMPLE:
> I want the title "Legacy Transformation Services" to have line-breaks so that the header html looks like `<h1>Legacy<br/>Transformation<br/>Services</h1>`
> To add a header with line-breaks, I need to add groups of 3 spaces to the title. So the title I would add to Contentful would be "Legacy   Transformation   Services".

I've just noticed that when this description is rendered it removes the 3-spacing... Sorry.

File: src/components/Contentful/Hero.js (hunk 1 above)
Comment (mmmmillar) [reply src/components/Contentful/Hero.js]: ok cool

**Pull request merged**